homeassistant/components/dynalite/__init__.py

```diff
@@ -139,7 +141,12 @@ def validate_area(config: Dict[str, Any]) -> Dict[str, Any]:
 
 AREA_SCHEMA = vol.Schema({num_string: vol.Any(AREA_DATA_SCHEMA, None)})
 
-PLATFORM_DEFAULTS_SCHEMA = vol.Schema({vol.Optional(CONF_FADE): vol.Coerce(float)})
+PLATFORM_DEFAULTS_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_FADE): vol.Coerce(float),
+        vol.Optional(CONF_QUERY_CHANNEL): num_string,
```

Comment:
These changes don't seem related to the service.

Reply:
when i implemented the service, i noticed that a couple of areas in my home don't return the preset and that is why i inserted the ability to choose which channel will be used to query the preset. i agree that it could be implemented separately, but the fix is really in the library. This just lets the user tell it also which channel to use for the automatic requests and not just the service calls.
do you want me to remove it and reimplement it in a different PR?

Reply:
also, it helped me in creating the unit-tests, as it solved a problem to me when they were together, but i will find another way to resolve it if they have to be separated

Reply:
Please make a separate PR for each thing.

Reply:
ok. separated the query_channel out. will create the PR for it after we finish with this to save me from complex merges. 
should be completely separate for the service at this point, so let me know if i forgot anything inside